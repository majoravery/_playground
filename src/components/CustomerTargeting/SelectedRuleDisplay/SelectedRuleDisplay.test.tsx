// import React from "react";
// import { render } from "@testing-library/react";
// import { head } from "ramda";
// import { SelectedRuleDisplay } from ".";
// import {
//   profileWithMultipleValuesMock,
//   profileWithSingleSelectionValueMock,
//   rulesConfigMock,
// } from "../__mocks__";
// import { CustomerTargetingRulesConfig } from "../types";

// const defaultProps = {
//   conditionKey: "",
//   conditionIndex: 0,
//   ruleKey: "",
//   options: [],
//   conditions: [],
//   values: {
//     profile: profileWithSingleSelectionValueMock,
//   },
//   addNewMode: false,
//   setOptions: jest.fn(),
//   setConditions: jest.fn(),
//   setAddNewMode: jest.fn(),
//   configs: (head(rulesConfigMock) as CustomerTargetingRulesConfig)?.rules || [],
// };

// describe("SelectedRuleDisplay", () => {
//   it("renders single tag", () => {
//     const { asFragment } = render(<SelectedRuleDisplay {...defaultProps} />);
//     expect(asFragment()).toMatchSnapshot();
//   });

//   it("renders multiple tags", () => {
//     const { asFragment } = render(
//       <SelectedRuleDisplay
//         {...defaultProps}
//         values={{ profile: profileWithMultipleValuesMock }}
//       />
//     );
//     expect(asFragment()).toMatchSnapshot();
//   });
// });
export {};
