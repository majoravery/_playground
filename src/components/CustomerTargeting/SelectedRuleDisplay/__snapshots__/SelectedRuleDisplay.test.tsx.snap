// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectedRuleDisplay renders multiple tags 1`] = `
<DocumentFragment>
  <div
    data-testid="selected-rule-display"
    style="margin-top: -1px; margin-bottom: -1px;"
  >
    <span
      class="small tag-container"
      tabindex="0"
    >
      <div
        class="tag-container--content"
      >
        android
      </div>
    </span>
    <span
      class="small tag-container"
      tabindex="0"
    >
      <div
        class="tag-container--content"
      >
        mweb
      </div>
    </span>
  </div>
</DocumentFragment>
`;

exports[`SelectedRuleDisplay renders single tag 1`] = `
<DocumentFragment>
  <div
    data-testid="selected-rule-display"
    style="margin-top: -1px; margin-bottom: -1px;"
  >
    <span
      class="small tag-container"
      tabindex="0"
    >
      <div
        class="tag-container--content"
      >
        android
      </div>
    </span>
  </div>
</DocumentFragment>
`;
